src/confcom/azext_confcom/README.md

```diff
@@ -643,3 +643,55 @@ This rule determines whether unencrypted writable storage from the UVM to the co
 ## allow_capabilities_dropping
 
 Whether to allow capabilities to be dropped in the same manner as allow_environment_variable_dropping.
+
+## Microsoft Azure CLI 'confcom katapolicygen' Extension Examples
+
+Run `az confcom katapolicygen --help` to see a list of supported arguments along with explanations. The following commands demonstrate the usage of different arguments to generate confidential computing security policies.
+
+**Prerequisites:**
+Install the Azure CLI and Confidential Computing extension.
+
+See the most recently released version of `confcom` extension.
+
+```bash
+az extension list-available -o table | grep confcom
+```
+
+To add the most recent confcom extension, run:
+
+```bash
+az extension add --name confcom
+```
+
+Use the `--version` argument to specify a version to add.
+
+Run this to update to the latest version if an older version is already installed:
+
+```bash
+az extension update --name confcom
+```
+
+The `katapolicygen` command generates confidential computing security policies using a kubernetes pod spec. You can control the format of the generated policies using arguments. Note: It is recommended to use images with specific tags instead of the `latest` tag, as the `latest` tag can change at any time and images with different configurations may also have the latest tag.
+
+**Examples:**
+
+Example 1: The following command creates a security policy and outputs it to the command line:
+
+```bash
+az confcom katapolicygen -y ./pod.yaml --print-policy
+```
+
+This command combines the information of images from the pod spec with other information such as mount, environment variables and commands from the pod spec to create a CCE policy.
```

Comment:
We may want to call it by the field that gets injected into the YAML here instead of "CCE policy"

Reply:
fixed, good catch